cmd/postgres.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/postgresConfig/get"
+	"github.com/supabase/cli/internal/postgresConfig/update"
+	"github.com/supabase/cli/internal/utils/flags"
+)
+
+var (
+	postgresCmd = &cobra.Command{
+		GroupID: groupManagementAPI,
+		Use:     "postgres",
+		Short:   "Manage Postgres database",
+	}
+	postgresConfigCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Manage Postgres database config",
+	}
```

Comment:
Can we use `postgres-config` instead for consistency with other commands? For eg. we currently have `network-bans` and `network-restrictions`.

Reply:
It was probably a mistake to have them be top-level commands - I can make it postgres-config, but you'll likely end up with a explosion of postgres related commands in the future

Reply:
Hmm there's also a discoverability issue if we nest the commands too deep. I'd prefer keeping it top level for now and refractor later when we add more. Either way not a deal breaker since it's marked experimental.